tests/gpu_dfcc/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_subdirectory(gpu_dfcc1)
+add_subdirectory(gpu_dfcc2)
```

Comment:
Would you humor me and just name them `gpudfcc1` or similar. No one can remember `-`/`_`, so I've banned the latter from test case names (plugin names can't be helped).